howso/feature_residuals.amlg

```diff
@@ -55,19 +54,9 @@
 			param_path
```

Comment:
This `param_path` variable/declaration may have been for when these residuals were stored in their own map. Check and remove if this is unused?

Reply:
still being used